src/pages/signup/__tests__/signup.component.test.js

```diff
@@ -0,0 +1,89 @@
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import wrapper from 'test/test-utils';
+
+import SignUp from '../signup.component';
+
+let mockIsLoading = false;
+
+jest.mock('service/signup', () => ({
+  useMutationSignup: () => ({
+    isLoading: mockIsLoading,
+  }),
+}));
+
+describe('<SignUp />', () => {
+  it('renders the title and all the inputs', () => {
```

Comment:
I think it is important to check if the "Sign In" link is being rendered as well, so my suggestion is to add an `expect` to check that.